bread/views/read.py

```diff
@@ -45,6 +50,48 @@ def get_required_permissions(self, request):
         return [f"{self.model._meta.app_label}.view_{self.model.__name__.lower()}"]
 
 
+class TableReadView(
+    BreadView,
+    PermissionRequiredMixin,
+    DjangoReadView,
+):
+
+    accept_global_perms = True
+    fields = ["__all__"]
+    urlparams = (("pk", int),)
+
+    def __init__(self, *args, **kwargs):
+        self.fields = expand_ALL_constant(
+            kwargs["model"], kwargs.get("fields") or self.fields
+        )
+        super().__init__(*args, **kwargs)
+
+    def get_layout(self):
+        return hg.BaseElement(
+            hg.H3(self.object),
+            _layout.datatable.DataTable(
+                columns=[
+                    _layout.datatable.DataTableColumn(header="", cell=hg.C("row.0")),
+                    _layout.datatable.DataTableColumn(header="", cell=hg.C("row.1")),
+                ],
+                row_iterator=(
+                    (
+                        field
+                        if isinstance(field, tuple)
+                        else (
+                            _layout.ObjectFieldLabel(field),
+                            _layout.ObjectFieldValue(field, formatter=format_value),
+                        )
+                    )
+                    for field in self.fields
+                ),
+            ),
+        )
+
+    def get_required_permissions(self, request):
+        return [f"{self.model._meta.app_label}.view_{self.model.__name__.lower()}"]
+
+
```

Comment:
This seems actually a much better option, especially for prototyping than using the readonly-form. It is also better readable.